_posts/2020-Q1-open-source-report.markdown

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: 2020-q1-open-source-report
+date: 2020-04-09 20:12:12
+categories: [open-source]
+author: bronzdoc
+
+---
+
+# Open Source at OmbuLabs: 2020 Q1 Report
+
+![ombulabs open source report](https://external-preview.redd.it/zbCp6e3VikD9vsKOtTGVKI90UnWIOZN5NMRNSvgvXxE.jpg?width=960&crop=smart&auto=webp&s=9318c01ae704565ad90f4736aec5af6b671d98fa)
```

Comment:
@bronzdoc Please download this image and put it in the assets. Also, this should be an html so we can have class and alt properties. for example:

`<img src="/assets/images/team_retreat/team-retreat-official.jpg" alt="team retreat ombulabs" class="medium-img">`